Translations/es.po

```diff
@@ -384,7 +384,7 @@ msgstr "Requisitos:"
 
 #: Source/DiabloUI/selconn.cpp:66
 msgid "no gateway needed"
-msgstr "sin puerta de enlace"
+msgstr "no se necesita puerta de enlace"
```

Comment:
The previous line barely fit, this one for sure won't :)

Reply:
OK. Discard this push then. I'll start a complete check of all the translation.

Reply:
```suggestion
msgstr "sin puerta de enlace"
```